README.md

```diff
@@ -296,13 +296,15 @@ In addition to the global configuration values above, when using E-Series, these
 | Option            | Description                                                               | Example       |
 | ----------------- | --------------------------------------------------------------------------| ------------- |
 | webProxyHostname  | Hostname or IP address of Web Services Proxy                              | localhost     |
+| webProxyPort      | Port number of the Web Services Proxy (optional)                          | 8443          |
+| webProxyInsecure  | Connect over http instead of https (optional, default = false)            | true          |
 | username          | Username for Web Services Proxy                                           | rw            |
 | password          | Password for Web Services Proxy                                           | rw            |
 | controllerA       | IP address of controller A                                                | 10.0.0.5      |
 | controllerB       | IP address of controller B                                                | 10.0.0.6      |
 | passwordArray     | Password for storage array if set                                         | blank/empty   |
 | hostData_IP       | Host iSCSI IP address (if multipathing just choose either one)            | 10.0.0.101    |
```

Comment:
Possibly change hostData_IP to hostDataIP to make it consistent with some of the other variable name changes you have done.